services/volunteer.service.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+const Volunteer = require("../models/volunteer.model");
+
+/**
+ * @function createVolunteer
+ * @param {JSON} volunteerDetails
```

Comment:
What's expected of volunteerDetails?

Reply:
Currently, all services follow this format. I will update them all in the future.

Reply:
When are you planning on updating them all?

Reply:
It's top priority.